api/app/signals/apps/api/v1/serializers/question.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+
+from signals.apps.signals.models import Question
+
+
+class PublicQuestionSerializerDetail(serializers.Serializer):
```

Comment:
Just Wondering why you used a "normal" Serializer instead of the ModelSerializer?

Reply:
2 reasons:
- there's no updating required only list
- future refactor will move away from the ModelSerializer (seperate layer)

Reply:
I can live with that, you @vanbuiten?

Reply:
@tcoenen @CBuiVNG 👍  ws just wondering why. Fine by me.